website/client/src/components/chat/autoComplete.vue

```diff
@@ -170,11 +163,26 @@ export default {
     this.grabUserNames();
   },
   methods: {
+    solveSearchResults (searchTextWip) {
+      if (!this.searchActive) return [];
+      const regexRes = this.atRegex.exec(searchTextWip);
+      if (!regexRes) return [];
+      this.currentSearch = regexRes[1]; // eslint-disable-line vue/no-side-effects-in-computed-properties, max-len, prefer-destructuring
+
+      const lowerCaseSearch = this.currentSearch.toLowerCase();
+
+      return this.tmpSelections
+        .filter(option => { // eslint-disable-line arrow-body-style
+          return option.displayName.toLowerCase().indexOf(lowerCaseSearch) !== -1
+            || (option.username
+              && option.username.toLowerCase().indexOf(lowerCaseSearch) !== -1);
+        })
```

Comment:
```suggestion
        .filter(option => option.displayName.toLowerCase().includes(lowerCaseSearch)
            || (option.username && option.username.toLowerCase().includes(lowerCaseSearch)))
```

Reply:
I think I maybe hitting the esline limit... gotta check